config/default.yml

```diff
@@ -711,3 +711,10 @@ Rails/TimeZone:
 Rails/Validation:
   Include:
     - app/models/**/*.rb
+
+##################### Performance ##################################
+
+Performance/Detect:
+  # Defines what method to use instead of `select.first`, e.g.
+  # `detect` or `find`.
+  PreferredMethod: detect
```

Comment:
No, not at all. It is impossible to prefer `find` over `detect` in general but still want `select.first` to be changed to `detect`. So it's a single concern, and specifying it in two places is a violation of the DRY principle.

Using another cop's configuration is something we do in a few places already. Search for `for_cop` in the source code.